momento-sdk/src/main/java/momento/sdk/ScsDataClient.java

```diff
@@ -408,6 +408,25 @@ CompletableFuture<CacheListConcatenateBackResponse> listConcatenateBackString(
     }
   }
 
+  CompletableFuture<CacheListConcatenateBackResponse> listConcatenateBackString(
+      String cacheName, String listName, List<String> values, int truncateFrontToSize) {
+    try {
```

Comment:
Can these just call the other signature with null?

Reply:
We can, I did this way because the set method creates different functions for handling null values and I wanted to be consistent with that. 
We can go either ways though. Do you want me to make the set methods without null to call set methods with null values?

Reply:
Got it, thanks. Will do.